framework/mesh/mesh.cc

```diff
@@ -15,297 +15,4 @@ GetCurrentMesh()
-        if (i == 0 and v == 0)
-          face.neighbor = 3 /*YMIN*/;
-
-        cell->faces.push_back(face);
-      }
-
-      umesh->AddCell(cell);
-    } // for j
-  }   // for i
-
-  umesh->ComputeCentroids();
-  umesh->CheckQuality();
-  umesh->BuildMeshConnectivity();
-
-  unpartitionedmesh_stack.push_back(umesh);
-
-  return unpartitionedmesh_stack.size() - 1;
-}
-
-size_t
-CreateUnpartitioned3DOrthoMesh(std::vector<double>& vertices_1d_x,
-                               std::vector<double>& vertices_1d_y,
-                               std::vector<double>& vertices_1d_z)
-{
-  OpenSnLogicalErrorIf(vertices_1d_x.empty() or vertices_1d_y.empty() or vertices_1d_z.empty(),
-                       "Empty vertex list.");
-
-  // Create unpartitioned mesh
-  auto umesh = std::make_shared<UnpartitionedMesh>();
-
-  umesh->SetDimension(3);
-
-  // Create vertices
-  size_t Nx = vertices_1d_x.size();
-  size_t Ny = vertices_1d_y.size();
-  size_t Nz = vertices_1d_z.size();
-
-  umesh->SetOrthoAttributes(Nx - 1, Ny - 1, Nz - 1);
-  umesh->AddBoundary(0, "XMAX");
-  umesh->AddBoundary(1, "XMIN");
-  umesh->AddBoundary(2, "YMAX");
-  umesh->AddBoundary(3, "YMIN");
-  umesh->AddBoundary(4, "ZMAX");
-  umesh->AddBoundary(5, "ZMIN");
-
-  // i is j, and j is i, MADNESS explanation:
-  // In math convention the i-index refers to the ith row
-  // and the j-index refers to the jth row. We try to follow
-  // the same logic here.
-
-  typedef std::vector<uint64_t> VecIDs;
-  typedef std::vector<VecIDs> VecVecIDs;
-  std::vector<VecVecIDs> vertex_ijk_to_i_map(Ny);
-  for (auto& vec : vertex_ijk_to_i_map)
-    vec.resize(Nx, VecIDs(Nz));
-
-  umesh->Vertices().reserve(Nx * Ny * Nz);
-  uint64_t c = 0;
-  for (size_t i = 0; i < Ny; ++i)
-  {
-    for (size_t j = 0; j < Nx; ++j)
-    {
-      for (size_t k = 0; k < Nz; ++k)
-      {
-        vertex_ijk_to_i_map[i][j][k] = c++;
-        umesh->Vertices().emplace_back(vertices_1d_x[j], vertices_1d_y[i], vertices_1d_z[k]);
-      } // for k
-    }   // for j
-  }     // for i
-
-  // Create cells
-  auto& vmap = vertex_ijk_to_i_map;
-  for (size_t i = 0; i < (Ny - 1); ++i)
-  {
-    for (size_t j = 0; j < (Nx - 1); ++j)
-    {
-      for (size_t k = 0; k < (Nz - 1); ++k)
-      {
-        auto cell =
-          new UnpartitionedMesh::LightWeightCell(CellType::POLYHEDRON, CellType::HEXAHEDRON);
-
-        cell->vertex_ids = std::vector<uint64_t>{vmap[i][j][k],
-                                                 vmap[i][j + 1][k],
-                                                 vmap[i + 1][j + 1][k],
-                                                 vmap[i + 1][j][k],
-
-                                                 vmap[i][j][k + 1],
-                                                 vmap[i][j + 1][k + 1],
-                                                 vmap[i + 1][j + 1][k + 1],
-                                                 vmap[i + 1][j][k + 1]};
-
-        // East face
```

Comment:
I also liked the comments about East face, West face, ..... do we still have then in the code that those lines are duplicates of?